apps/admin/pages/api/test.tsx

```diff
@@ -0,0 +1,194 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { apiHandler } from "../../middlewares/apiHandler";
+import { Prisma } from "@prisma/client";
+import { getDMMF } from "@prisma/internals";
+
+export default apiHandler(
+  async (req: NextApiRequest, res: NextApiResponse, auth) => {
+    const tables = Object.keys(Prisma.ModelName);
+    const dmmf = await getDMMF({
+      datamodel: `// This is your Prisma schema file,
```

Comment:
the schema as text lives somewhere in the client (prismaClient) super sure I have seen it